### PR TITLE
Messenger: resize routine redux

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Resizes the ad slot. This will be used by expandable creatives.

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/messenger/resize.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+], function (assign, closest, fastdom, messenger) {
+    messenger.register('resize', function(specs, ret, iframe) {
+        return resize(specs, closest(iframe, '.js-ad-slot'));
+    });
+
+    return resize;
+
+    function resize(specs, adSlot) {
+        if (!specs || !('height' in specs || 'width' in specs)) {
+            return null;
```
Comment (rich-nguyen) [inline static/src/javascripts/projects/commercial/modules/messenger/resize.js]: `Promise.resolve()` ?

**Comment by rich-nguyen** [comment]
👍

File: static/src/javascripts/projects/commercial/modules/messenger/resize.js (hunk 1 above)
Comment (regiskuckaertz) [inline static/src/javascripts/projects/commercial/modules/messenger/resize.js]: The routine is already wrapped in a promise callback 😄

Comment (rich-nguyen) [inline static/src/javascripts/projects/commercial/modules/messenger/resize.js]: oh!

**Pull request merged**